File: MyTournamentApp/app/src/main/java/com/example/ourtournament/Objetos/TorneoSeguido.java
package com.example.ourtournament.Objetos;

public class TorneoSeguido extends Torneo{
    public Boolean Siguiendo;
    public TorneoSeguido(int idtorneo, String nombreTorneo, String contraseniaDeAdministrador, String linkParaUnirse, boolean siguiendo)
    {
        super(idtorneo, nombreTorneo,  contraseniaDeAdministrador, linkParaUnirse);
        Siguiendo = siguiendo;
    }

    public TorneoSeguido()
    {
        this(0,"","","",false);
    }
}

File: MyTournamentAPI/OurTournamentAPI/Models/TorneoSeguido.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace OurTournamentAPI.Models
{
    public class TorneoSeguido:Torneo
    {
        private Boolean siguiendo;

        public TorneoSeguido(int idtorneo, string nombreTorneo, string contraseniaDeAdministrador, string linkParaUnirse, bool siguiendo) :
            base( idtorneo,  nombreTorneo,  contraseniaDeAdministrador,  linkParaUnirse)
        {
            this.siguiendo = siguiendo;
        }
        public TorneoSeguido() : this(0, "", "", "", false)
        {
        }
        public bool Siguiendo { get => siguiendo; set => siguiendo = value; }
    }
}